sphinx/registry.py

```diff
@@ -125,6 +127,9 @@ def __init__(self) -> None:
         #: additional transforms; list of transforms
         self.transforms: List[Type[Transform]] = []
 
+        # private cache of Docutils Publishers (file type -> publisher object)
```

Comment:
Basically, we don't use the `_` prefix for attribute names. From our experience, private markers are ignored by hackers. So we need to take care of both public and private methods.

Reply:
Done